test/purgecache_test.js

```diff
@@ -1,35 +1,29 @@
-suite('Purge-Cache', () => {
-  let helper = require('./helper');
-  let assume = require('assume');
+const helper = require('./helper');
+const assume = require('assume');
+
+helper.secrets.mockSuite(helper.suiteName(__filename), ['azure'], function(mock, skipping) {
```

Comment:
I *think* this will fail if given azure secrets but not taskcluster secrets, no?